x-pack/plugins/canvas/public/components/app/index.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { History } from 'history';
+// @ts-expect-error
+import createHashStateHistory from 'history-extra/dist/createHashStateHistory';
+import { useServices } from '../../services';
+// @ts-expect-error
+import { shortcutManager } from '../../lib/shortcut_manager';
+import { CanvasRouter } from '../../routes';
+
+class ShortcutManagerContextWrapper extends React.Component {
```

Comment:
This should be refactored soon, too.

https://reactjs.org/docs/legacy-context.html

Reply:
Yep, the `ReactShortcuts` package we use uses the legacy context, hence the need for it here.  That package is also not really maintained, so may want to look into other replacements that are more up to date.